ui/src/store/modules/app.ts

```diff
@@ -9,12 +9,14 @@ export interface Message {
 }
 
 export interface AppState {
+  language: string
   messages: Message[]
   loading: boolean
   commonRDFProperties: string[]
 }
 
 const initialState = {
+  language: 'en',
```

Comment:
Might try [navigator.language](https://caniuse.com/mdn-api_navigatorlanguage_language) to hope for user's agent language

Reply:
I don't think that's worth it, since the whole app is in english anyway